src/wacks4square/square.py

```diff
@@ -116,10 +116,24 @@ def get_sales_since_timestamp(self, timestamp: Optional[str]) -> List[Sale]:
         new_orders_response = self._get_orders_since_timestamp(timestamp)
         sales = []
         for order in new_orders_response:
+            order_id = order.get("id")
             line_items = order.get("line_items", [])
+            if not line_items and order.get("refunds"):
+                logger.warning(f'refund order, order id {order_id}')
             for item in line_items:
                 listing_id = item.get("catalog_object_id")
-                if not listing_id:
+                item_name = item.get("name", "")
+                # this is used as a fallback name for the sale if we don't find it by id in werbies.json
+                fallback_name = f'{item_name} ({item.get("variation_name", "")})'
```

Comment:
maybe something like this to remove when there'd be empty parantheses if there is no variation?

```suggestion
                variation = f"({item.get("variation_name")})" if item.get("variation_name") else ""
                fallback_name = f'{item_name} {variation}'
```